spec/models/company_spec.rb

```diff
@@ -2,31 +2,44 @@
 
 describe Company do
   describe "validations" do
-    context "invalid attributes" do
-      it "is invalid without a name" do
-        company = Company.new()
-        expect(company).to be_invalid
-      end
-
-      it "has a unique name" do
-        Company.create(name: "Dropbox")
-        company = Company.new(name: "Dropbox")
-        expect(company).to be_invalid
-      end
+    context "it must have a unique name" do
+      it {should validate_uniqueness_of(:name)}
     end
 
     context "valid attributes" do
-      it "is valid with a name" do
-        company = Company.new(name: "Dropbox")
-        expect(company).to be_valid
-      end
+      it {should validate_presence_of(:name)}
     end
   end
 
   describe "relationships" do
-    it "has many jobs" do
-      company = Company.new(name: "Dropbox")
-      expect(company).to respond_to(:jobs)
+    context "has many jobs" do
+      it {should have_many(:jobs)}
+    end
+
+    context "has many contacts" do
+      it {should have_many(:contacts)}
+    end
+  end
+
+  describe "#methods" do
+    before do
+      FactoryGirl.reload
+      create(:company_jobs)
+    end
+
+    it "average interest" do
```

Comment:
we probably want a more specific test for this method - we want to make sure the average interest is accurate